### DISCUSSION TITLE
Remote for Echostar HDS-600rs

### DISCUSSION THREAD

**Discussion by BobC1954** [GerryDazoo/Slinger#189] [General]
Here is what I cobbled up, thanks to Gerry. I have not included the controls that were dedicated to the IR TV function. 
[HDS600rs.txt](https://github.com/GerryDazoo/Slinger/files/9891420/HDS600rs.txt)

**Comment by GerryDazoo** [comment]
THX. The next version with have a working remote for the TV tuner. At least ch+ ch- Last and direct tuning.

**Comment by RandomMan719** [comment]
I used this remote (shawcable.xfinity.comcastX1.XR5.remote.txt) and made a couple changes, to make a custom remote for the Pace RNG200N, Digital Cable box.  I had to change the "Last" code the "Menu" code and I made the "Search" button, a "Power On/Off" button.  Doesn't have every button and feature, but it is enough for me.
[PACE_RNG200N_Basic.txt](https://github.com/GerryDazoo/Slinger/files/9892035/PACE_RNG200N_Basic.txt)

Thanks for working on this project.
